styles.css

```diff
@@ -36,43 +39,227 @@ h1 {
   font-weight: 400;
   font-size: 14px;
   line-height: 21px;
-  margin-top: 10px;
+  margin: 20px 0;
   max-width: 500px;
   color: #1a2236;
 }
 
-.collab-button {
+.button {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-family: "Poppins", sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
   text-decoration: none;
   max-width: 500px;
   height: 50px;
-  margin: 30px 0;
+  margin: 45px 0 20px 0;
+  width: 100%;
+  background: #1a2236;
+  color: #fff;
 }
 
-.button {
+.button:hover {
+  background: #fff;
+  color: #1a2236;
+}
+
+.bottom-img {
   display: flex;
   flex-direction: column;
+  align-items: center;
+}
+
+.bg-bottom {
+  width: 100%;
+  max-width: 100%;
+}
+
+.proj-section {
+  display: grid;
+  grid-template-columns: auto;
+  margin-top: auto;
+}
+
+h2.heading {
+  margin: 40px 0 32px 16px;
+}
+
+h2.about {
+  margin: 51px 0 20px 0;
+}
+
+.proj-img {
+  width: 100%;
+  max-width: 100%;
+  margin: -5px 0 -10px 0;
+}
+
+.proj-name {
+  padding: 30px 0 0 20px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-size: 20px;
+  font-weight: 300;
+  line-height: 80%;
+}
+
+.proj-description {
+  background: #1a2236;
+  color: #fff;
+  padding-bottom: 25px;
+}
```

Comment:
- [ ] Good job on making  a good design, :1st_place_medal: :1st_place_medal: however, as requested by the previous code reviewer, I think it would be better to make content of the ```.proj-description``` class aligned with the ```flexbox``` to meet the project requirements :+1:

Reply:
Thank you so much. Your review prompted me to think of small loopholes to improve my code.